read-fonts/src/tables/variations.rs

```diff
@@ -661,6 +688,7 @@ impl<'a, T> TupleVariationIter<'a, T>
 where
     T: TupleDelta,
 {
+    #[inline(never)]
```

Comment:
I'm curious about what inline(never) is buying us?

Reply:
see the explanation above